src/adapters/interport/execution/src/index.ts

```diff
@@ -0,0 +1,10 @@
+import { UserBalance } from './sdk/types';
+import {
+  getUserPositionsAtBlock,
+} from './sdk/lib';
+
+export const getUserBalanceByBlock = async (blockNumber: number, blockTimestamp: number): Promise<UserBalance[]> => {
```

Comment:
The function signature has been modified; please use `function getUserTVLData(blockNumber: number)`.

Reply:
Done